plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java

```diff
@@ -52,4 +52,11 @@ public void applyNotationForEntities(
         @NotNull ERDAssociation association,
         @NotNull Color bckColor,
         @NotNull Color frgColor);
+
+    /**
+     * Method designed to provide specific value of indentation
+     *
+     * @return - value of indentation
+     */
+    public double getIndentation();
```

Comment:
Indentation between what?

Reply:
Can you clarify that in the method's name?

Reply:
The suitable place for it is a java doc.